AsyncDisplayKit/ASImageNode.mm

```diff
@@ -110,17 +109,11 @@ - (instancetype)initWithViewBlock:(ASDisplayNodeViewBlock)viewBlock didLoadBlock
   return nil;
 }
 
-- (void)dealloc
```

Comment:
We need to add that back in! That was a refactor to remove the dealloc from the `ASImageNode` category and move it into `ASImageNode`